### PR TITLE
#68 fixing import directives

### PR DESCRIPTION
**Pull request by Sealdolphin**
- Changed name of import libraries

### CODE REVIEW

File: source/incqueryserver-jupyter/iqs_jupyter/analysis_extensions.py [hunk 1]
```diff
@@ -19,7 +19,7 @@
 
 import html
 
-import iqs_client
+from iqs_client import models as iqs_client
```
Comment (FuzesiMate) [inline source/incqueryserver-jupyter/iqs_jupyter/analysis_extensions.py]: `iqs_client` is the import of the root module, do not use the same alias for the `models` submodule

**Review by FuzesiMate — CHANGES_REQUESTED** [review]

File: source/incqueryserver-jupyter/iqs_jupyter/__init__.py [hunk 2]
```diff
@@ -30,18 +30,15 @@
 from iqs_jupyter.core_extensions import *
 from iqs_jupyter.authentication import *
 
-if 'MmsRepositoryApi' in dir(iqs_client):
-    from iqs_jupyter.mms_extensions import *
-    from iqs_jupyter.mms_direct_extensions import *
+#if 'MmsRepositoryApi' in dir(iqs_client):
```
Comment (bergmanngabor) [inline source/incqueryserver-jupyter/iqs_jupyter/__init__.py:34]: I understand that we no longer ship trimmed down versions of the core API, but is there any specific need to remove these conditionals? I would expect them to work just fine.

File: source/incqueryserver-jupyter/iqs_jupyter/analysis_extensions.py [hunk 3]
```diff
@@ -19,13 +19,14 @@
 
 import html
 
-import iqs_client
```
Comment (bergmanngabor) [inline source/incqueryserver-jupyter/iqs_jupyter/analysis_extensions.py:22]: Why was this change necessary? Was there a problem with polluting the namespace?
Issue description #68 tells me nothing.
I do not understand WHY we are changing these.

File: source/incqueryserver-jupyter/iqs_jupyter/core_extensions.py [hunk 4]
```diff
@@ -22,7 +22,17 @@
...
+from iqs_client.models import ElementInCompartmentDescriptor,\
+    TypedElementInCompartmentDescriptor,\
+    QueryExecutionResponse,\
+    ModelCompartment,\
+    IndexMessage,\
+    SimpleMessage,\
+    QueryFQNList, \
+    GenericValidationResults,\
+    GenericValidationRule, \
+    ValidationDiagnostics
```
Comment (bergmanngabor) [inline source/incqueryserver-jupyter/iqs_jupyter/core_extensions.py]: such long import blocks sound very uncomfortable and brittle. Is there a specific reason that we cannot rely on `iqs_client.models`, perhaps with the alias `schema`?

**Review by bergmanngabor — COMMENTED** [review]
Since I have no idea what the problem is, I cannot judge how well it is being solved here.

File: source/incqueryserver-jupyter/iqs_jupyter/__init__.py (hunk 2 above)
Comment (Sealdolphin) [reply source/incqueryserver-jupyter/iqs_jupyter/__init__.py:34]: I found it inconsistent, because the new generator creates a different architecture... If we want it to stay, we may change it, but I thought it was easier for now to omit these features. This is a work in progress.

File: source/incqueryserver-jupyter/iqs_jupyter/analysis_extensions.py (hunk 3 above)
Comment (Sealdolphin) [reply source/incqueryserver-jupyter/iqs_jupyter/analysis_extensions.py:22]: Yes it seems, that the generated client has been changed. And `iqs_client` no longer has the necessary DTO-s. (The decorator may not run, that is why.) My initial goal is to be consistent with the release, however we may need to change the python client as well.

File: source/incqueryserver-jupyter/iqs_jupyter/core_extensions.py (hunk 4 above)
Comment (Sealdolphin) [reply source/incqueryserver-jupyter/iqs_jupyter/core_extensions.py]: We absolutely can use `iqs_client.models`, yes.

Comment (bergmanngabor) [reply source/incqueryserver-jupyter/iqs_jupyter/core_extensions.py]: Note that the package `iqs_jupyter` already exposes this Python module as `schema`, and I believe most notebooks use something like `from iqs_jupyter import schema` and then `schema.ComputeCompartmentDeltaRequest` etc. We could similarly use the alias `schema` here.

File: source/incqueryserver-jupyter/iqs_jupyter/__init__.py (hunk 2 above)
Comment (bergmanngabor) [reply source/incqueryserver-jupyter/iqs_jupyter/__init__.py:34]: Have we changed which openapi generator we use? Is there a specific reason why we instituted such a change? Does it otherwise generate better client code?

Comment (Sealdolphin) [reply source/incqueryserver-jupyter/iqs_jupyter/__init__.py:34]: Yes, the new version is generated using a gradle plugin, instead of a library.

File: source/incqueryserver-jupyter/iqs_jupyter/core_extensions.py [hunk 5]
```diff
@@ -224,7 +224,7 @@ def _monkey_patch_generic_response_message_repr_html_(self):
     return '<span title="{}">{} <i>(see hover for details)</i></span>'.format(html.escape(self.to_str()), html.escape(self.message))
 
 def _monkey_patch_query_fqn_list_repr_html_(self):
-    ns_list = self.query_fq_ns
+    ns_list = self.query_fqns
```
Comment (Sealdolphin) [inline source/incqueryserver-jupyter/iqs_jupyter/core_extensions.py:227]: This is required because of the changes in the generated code

File: source/incqueryserver-jupyter/iqs_jupyter/__init__.py (hunk 2 above)
Comment (bergmanngabor) [reply source/incqueryserver-jupyter/iqs_jupyter/__init__.py:34]: No, I mean OpenAPI (whether invoked from the CLI or via Gradle) has a selection of plugins called "generators". Are we still using the same one?

**Review by bergmanngabor — APPROVED** [review]

File: source/incqueryserver-jupyter/iqs_jupyter/__init__.py (hunk 2 above)
Comment (FuzesiMate) [reply source/incqueryserver-jupyter/iqs_jupyter/__init__.py:34]: @bergmanngabor - We would like to upgrade OpenApi generator to 5.x project-wise in IQS, I think it is reasonable to use newer version, because later we might face issues caused by the outdated generator.

**Comment by Sealdolphin** [comment]
It seems, that out Open API generator causes many issues in the client, which may break this extension. Therefore we do not upgrade the Open API version for now. The old version (4.2.3.) stays instead.

This means, that this issue becomes irrelevant.
